modules/rosa-hcp-upgrading-cli-machinepool.adoc

```diff
@@ -41,19 +42,19 @@ Machine pool configurations such as node drain timeout, max-unavailable, and max
 +
 [source,terminal]
 ----
-$ rosa describe cluster --cluster=<cluster_name_or_id> <1>
+$ rosa describe cluster --cluster=<cluster_name_or_id>
 ----
-<1> Replace `<cluster_name_or_id>` with the cluster name or the cluster ID.
+Replace `<cluster_name_or_id>` with the cluster name or the cluster ID.
 +
 ifeval::["{context}" != "rosa-hcp-upgrading-whole-cluster"]
-.Example output
+*Example output*
```

Comment:
You need to add a `+` to fix this:

<img width="914" height="388" alt="Image" src="https://github.com/user-attachments/assets/08371672-0fa3-4036-b918-a9b59e786c6f" />